test/app/App.js

```diff
@@ -867,7 +820,7 @@ describe('App', function() {
 		});
 	});
 
-	it('should restore scroll position on navigate back', (done) => {
+	xit('should restore scroll position on navigate back', (done) => {
```

Comment:
Why is this one disabled? Can we either fix, or delete, or scope to run not on IE etc? If none of the above applies, a comment indicating why would be helpful.

Reply:
Ok. I think this one was disabled because it was acting "flaky" at some point, but I re-enabled those 3 disabled tests and am waiting for Travis to see how this will behave, dependending on the result, I'll see what our best option is.